<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-S76SA9N" failures="1" tests="4" name="com.nexsoft.test.CreateEventTest" time="35.594" errors="0" timestamp="2022-06-23T00:31:16 ICT" skipped="1">
  <testcase classname="com.nexsoft.test.CreateEventTest" name="createEvent1" time="16.507"/>
  <system-out/>
  <testcase classname="com.nexsoft.test.CreateEventTest" name="createEvent2" time="17.482"/>
  <system-out/>
  <testcase classname="com.nexsoft.test.CreateEventTest" name="getEventName" time="1.605">
    <failure message="Lists differ at element [0]: Ã°Å¸â€œâ€¦   Sparing1 != ðŸ“…   Sparing1 expected [Ã°Å¸â€œâ€¦   Sparing1] but found [ðŸ“…   Sparing1]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Lists differ at element [0]: Ã°Å¸â€œâ€¦   Sparing1 != ðŸ“…   Sparing1 expected [Ã°Å¸â€œâ€¦   Sparing1] but found [ðŸ“…   Sparing1]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:1479)
at org.testng.Assert.assertEquals(Assert.java:1441)
at com.nexsoft.test.CreateEventTest.getEventName(CreateEventTest.java:99)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- getEventName -->
  <system-out/>
  <testcase classname="com.nexsoft.test.CreateEventTest" name="createEventTest">
    <skipped/>
  </testcase> <!-- createEventTest -->
  <system-out/>
</testsuite> <!-- com.nexsoft.test.CreateEventTest -->
